Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/VanillaMemeEdit.java

```diff
@@ -22,6 +38,40 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    public void onShareClick(View v){
```

Comment:
Looks good. I agree with line 59 in the share click method...do you need help implementing this ?